data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -1,12 +1,10 @@
 #textdomain wesnoth-multiplayer
 
-# original map by telex4, recreated by Becephalus 1/06, revised bex 05/06
-
 [multiplayer]
     id=multiplayer_King_of_the_Hill
     name= _ "4p — King of the Hill"
     map_file=multiplayer/maps/4p_King_of_the_Hill.map
-    description= _ "Controlling the area around the central keep is very lucrative in this 4 player FFA map. Works fine 2 vs. 2 as well. There are 36 villages." + _ " Recommended setting of 2 gold per village and 150 starting gold."
+    description= _ "The King is dead. Without an heir, his quarrelsome lords now vie for the hilltop throne and all the power and wealth it commands. Are you mighty enough to take this seat of power before your rivals do? Can you hold out against their many assassins? Is your arse truly great enough to fill the throne of a king? This 4 player FFA map works fine 2 vs. 2 as well. There are 38 villages and bonus income for the leader on the center keep." + _ " Recommended setting of 2 gold per village and 150 starting gold."
```

Comment:
Suggestion:

> This 4 player FFA map works fine 2 vs. 2 as well.

"This map can be played as a free for all or 2v2.

Reply:
Does that mean you're okay with the use of "arse" in here?

Reply:
Suggestion applied, thanks.

Reply:
It _is_ more of a British term, I believe, but as long as you're okay with it…

Reply:
It did seem out of character compared to the rest of the game, but as long as we don't keep adding more and more profanity laced descriptions to things I don't think it's an issue. It definitely gives it a more amusing/interesting feel.

Reply:
I only just noticed this while reviewing the translation files. I did wonder about its acceptability till I saw this conversation. I was also wondering about the British spelling - I suppose it must somehow look less vulgar than the American spelling? (Which I tend to read as 'female donkey' anyway...)

Reply:
…I thought it just meant "donkey", not specifically a female?

Reply:
You're probably right.

Reply:
> I was also wondering about the British spelling

America and other overseas colonies did not really exist in the pre-renaissance time period which BfW emulates. Maybe it is just me, but I find reading or hearing such modern dialects in a dark age or medieval setting to be jarring and mood breaking. So paying at least some lip service to period and setting "authenticity" felt somehow appropriate.

Anyway though, that was the only reason for the spelling.

Reply:
Ah, I forgot about the historical aspect. Same reasoning for why it's the Sceptre of Fire, not scepter as those following American conventions usually spell the word.

Reply:
> America and other overseas colonies did not really exist in the pre-renaissance time period which BfW emulates. Maybe it is just me, but I find reading or hearing such modern dialects in a dark age or medieval setting to be jarring and mood breaking. So paying at least some lip service to period and setting "authenticity" felt somehow appropriate.

That would be an argument in favour of using American or Canadian English though, as they're less diverged from the historical dialect that would've been spoken in 15th or 16th century Britain – in particular, unlike Britain they retained the R sound after most vowels. (Non-rhotic English was just starting to become a thing around the colonization era; it wasn't considered standard until around the 18th century.)

Reply:
I don't think you can make a blanket statement one way or another - it would be dependent on the specifics of a given variation. For example, I stayed with -ize suffixes for `en_GB` because that's the convention the previous maintainers followed and I suppose it follows the rationale that -ise suffixes is a newer convention. But there are lot of aspects of American English I would be surprised if they weren't very modern constructs.

Anyway, I was only querying this because I was initially surprised at the vulgarity introduced here (though I appreciate prose for MP is often more light-hearted and less formal that for SP). And I did find it mildly amusing though I usually wouldn't use the word myself. I wasn't intending to start another prolonged discussion on the nuances of language - some people might complain about being 'spammed' again.

Reply:
I'd say there's definitely a difference between debating something in a single comment thread with a few comments a day vs hitting 200 comments in under 2 days. If anyone thinks something like this comment thread is spam, well then that's their problem.

Reply:
> But there are lot of aspects of American English I would be surprised with if they weren't very modern constructs.

True enough. I was merely pointing out one way that _British_ English has changed since the colonization era, as a way to indicate that it's wrong to assume British English is "older". Neither American nor British English is the dialect spoken in the colonization era – they've both evolved in different directions from there.

Reply:
> in particular, unlike Britain they retained the R sound after most vowels.

That is true, but only of the spoken form. The "silent" R's still appear in the spelling, and since there is no voice acting in Wesnoth, it does not act as an anachronism.